runners/spark/src/main/java/org/apache/beam/runners/spark/io/ConsoleIO.java

```diff
@@ -31,17 +32,27 @@ private ConsoleIO() {
 
   /**
    * Write on the console.
+   *
+   * @see StreamingTransformTranslator#print()
```

Comment:
`StreamingTransformTranslator#print()` is a **private** method pertaining to the implementation of this transform, I would therefore avoid specifying it in the **public** javadoc aimed at telling users *what* this transform does, rather than *how*.